### DISCUSSION TITLE
Announcing the winners of MindsDB 2022 Hacktoberfest competition

### DISCUSSION THREAD

**Discussion by C0staTin** [mindsdb/mindsdb#4077] [Announcements]
![hacktoberfest_1a_1,91x1_1200x628](https://user-images.githubusercontent.com/12409467/202799659-165ba330-7c9b-4f79-9815-bf1cefa45852.jpg)

Hi everyone! :wave:

We are super-excited :smiley: to share the winners:trophy: of the [MindsDB Hacktoberfest](https://mindsdb.com/hacktoberfest) 2022 Prize Draw!
And the main prize ([Razer Blade 15 Laptop](https://www.razer.com/hard-bundles/Razer-Blade-15-Bundle-V5-Essential/RZHB-220912-11)) goes to....

:tada::trophy: [Rutam Prita Mishra](https://github.com/Rutam21) :trophy::tada:

Wow!!! Congratulations!!! :champagne::confetti_ball:

But that's not all!
We have ten extra cool gadgets (you can choose between JBL Earbuds or Rugged Outdoor Speaker). And, the winners are....
1. [Zubeen](https://github.com/syedzubeen)
2. [Alissa Troiano](https://github.com/alissatroiano)
3. [Hridya Agrawal](https://github.com/hridaya423)
4. [Michael Lantz ](https://github.com/MichaelLantz)
5. [HellFire](https://github.com/Artemis6969)
6. [Teslim Odumuyiwa](https://github.com/Tes-program)
7. [Anamika](https://github.com/noviicee)
8. [Arman Chand](https://github.com/ArmanChand)
9. [Akhil](https://github.com/akhildevelops)
10. [alexandre-dz-oscore](https://github.com/alexandre-dz-oscore)

Congratulations!!! :tada::champagne:

Here's the video of the prize draw to enjoy!
https://www.youtube.com/watch?v=cpGWlu7Fipk

Again, that's not all :-)
We have 2 more hacktoberfest prize categories.

The winners of the ideas 💡 contest are:

1. 1st place: [@Arman](https://mindsdbcommunity.slack.com/team/U03KHE55HV5) who suggested [integration with NoSQL Database Aerospike](https://github.com/mindsdb/mindsdb/issues/3273) :first_place_medal:
2. 1st place: [@Balaji Seetharaman](https://mindsdbcommunity.slack.com/team/U045AQHL83Z) who suggested  [integration with Atlassian Jira](https://github.com/mindsdb/mindsdb/issues/3676) :first_place_medal:
3. 3rd place: [@Rutam Prita Mishra](https://mindsdbcommunity.slack.com/team/U02H21VND4N) who suggested [integration with RethinkDB](https://github.com/mindsdb/mindsdb/issues/3274) :third_place_medal:

And the winners of the video 📽️ tutorials contest are:

1. 1st place: [@Sumit](https://mindsdbcommunity.slack.com/team/U046QG87X50) with the video [What is MindsDB - AI Database Prediction](https://www.youtube.com/watch?v=8ySq7Y3sfJs)
2. 2nd place: [@Bibek Bhatta](https://mindsdbcommunity.slack.com/team/U045XJ91KDF) with the video [Add AI into your Database with MindsDB. MindsDB Tutorial. How to get started Guide?](https://www.youtube.com/watch?v=z7tRc8JpuRI)
3. 3rd place: [@Hritik](https://mindsdbcommunity.slack.com/team/U0464QPF5K6) with the video [What is MindsDB | How to Get Started | A Cloud Database](https://www.youtube.com/watch?v=zHReFuFUI34)

Again, :pray:Big Thanks:pray: to all contributors and members of our amazing community who submitted ideas and voted!
Stay tuned for the next contests and cool prizes :wink:

**Comment by Rutam21** [comment]
I am really humbled to have won the Grand Prize and the 2nd Runners Up in Ideation challenge. A big thank you to the whole MindsDB team for providing such a great opportunity during last year's Hacktoberfest. Looking forward to this year's as well. 🎉
